python_utils.py

```diff
@@ -430,3 +430,17 @@ def yaml_from_dict(dictionary, width=80):
     """
     dictionary = _recursively_convert_to_str(dictionary)
     return yaml.safe_dump(dictionary, default_flow_style=False, width=width)
+
+
+class ChangeDirectory(OBJECT):
```

Comment:
This shouldn't be here -- python_utils is for things that are different in py2/py3.

There is already a CD class in common.py.